superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/ag-grid/AGGridViz.tsx

```diff
@@ -448,6 +450,28 @@ export default function AGGridViz({
           }
           key="submit-file-to-assembly-line"
           data={selectedData.typeData.eml_path}
+          base_url={assemblyLineUrl}
+          disabled={selectedData.typeData.eml_path.length > SUBMISSION_LIMIT}
```

Comment:
I like that you check the number of distinct paths for the tooltip. Should we do the same here?

Reply:
Revised code to only submit distinct eml paths in multiselect